packages/web/src/components/edit-track/components/PreviewButton.tsx

```diff
@@ -21,9 +21,7 @@ export const PreviewButton = (props: PreviewButtonProps) => {
   const { index, className } = props
   const { playingPreviewIndex, togglePreview } =
     useContext(UploadPreviewContext)
-  const [{ value: track }] = useField<CollectionTrackForUpload>(
-    `tracks.${index}`
-  )
+  const [{ value: track }] = useField<TrackForUpload>(`tracks.${index}`)
```

Comment:
calling out for @dylanjeffers - changed type here bc in this file we don't need the `override` field that `CollectionTrackForUpload` gives us.